package.json

```diff
@@ -14,6 +14,6 @@
   },
   "devDependencies": {
     "husky": "^8.0.3",
-    "turbo": "^1.9.6"
+    "turbo": "^1.10.12"
```

Comment:
Why do you need to to update `turbo`?
Is it required to let this functionality work?

Reply:
it does not work on arm mac os otherwise

Reply:
@tommaso1 Ok, maybe in future PR's I'd add this kind of information within the PR's template to share it with the whole team.

Anyway, IMHO I see two options:
- Create a separate PR with the bump of turbo, explaining why it's needed
- Bump turbo here and it would be nice if you add something more about this into the PR's template

Please note that if you follow Option 1, then [you must update `package-lock.json` file into this PR](https://github.com/pagopa/developer-portal/actions/runs/5724798142/job/15512020098?pr=218#step:4:5)